nautobot/core/views/renderers.py

```diff
@@ -184,6 +185,9 @@ def get_context(self, data, accepted_media_type, renderer_context):
                 form = form_class(initial=request.GET)
             elif view.action in ["create", "update"]:
                 initial_data = normalize_querydict(request.GET, form_class=form_class)
+                # Manually populate the reverse side of the manytomany relationship in ContactForm
+                if isinstance(instance, Contact):
+                    initial_data["teams"] = instance.teams.all()
                 form = form_class(instance=instance, initial=initial_data)
```

Comment:
Could we instead of implementing this in core, implement it in the `ContactForm` class `__init__`?

Reply:
That's what we discussed yesterday in DSU. Not sure if @HanlinMiao has tried it yet